packages/fxa-auth-server/lib/routes/password.ts

```diff
@@ -108,7 +108,7 @@ module.exports = function (
           await customs.checkAuthenticated(
             request,
             sessionToken.uid,
-            'passwordChange'
+            'authenticatedPasswordChanged'
```

Comment:
That's not a defined action in the customs server.

Reply:
I'll add v2 check here and fallback accordingly.